src/lib/db/index.ts

```diff
@@ -95,10 +137,16 @@ export default class Database {
         [name]: () => this.client(name),
         ...acc,
       }
-    }, {}) as Models<() => Query>
+    }, {}) as Models<() => QueryBuilder>
```

Comment:
🤦 used a wrong type

Reply:
No more I just needed to do some renaming so I could use the `Query` from `tsoa`. I felt this was a better name regardless